packages/text-area/src/TextArea/TextArea.spec.tsx

```diff
@@ -20,14 +17,33 @@ const defaultProps = {
 };
 
 function renderTextArea(props = {}) {
-  const utils = render(
-    <TextArea data-testid="text-area" label={labelProp} {...props} />,
-  );
-
-  const textArea = utils.getByTestId('text-area');
-  const label = utils.container.querySelector('label');
-  const description = utils.container.querySelector('p');
-  return { ...utils, textArea, label, description };
+  const renderUtils = render(<TextArea label={labelProp} {...props} />);
```

Comment:
What happens when this is a form with multiple text elements?

Reply:
I have tests for this if you scroll more down to `getLGTextAreaUtils.spec.tsx`

Reply:
ah amaaizng thank you